test/powershell/Modules/Microsoft.PowerShell.Utility/Format-List.Tests.ps1

```diff
@@ -34,9 +34,9 @@ Describe "Format-List" -Tags "CI" {
 
     It "Should be able to call a property of the piped input" {
         # Tested on two input commands to verify functionality.
-        { Get-Command | Select-Object -First 5 | Format-List -Property Name }        | Should -Not -BeNullOrEmpty
+        Get-Command | Select-Object -First 5 | Format-List -Property Name | Should -Not -BeNullOrEmpty
```

Comment:
This is deceptive.  It seems like you are verifying you get 5 commands but only verify you get one.  You might as well change `5` to `1` for clarity

Reply:
Changed 5 to 1.